tests/writer.rs

```diff
@@ -407,4 +407,26 @@ mod tests {
             fs::read("resources/rgb/4D.nii").unwrap()
         );
     }
+
+    #[test]
+    fn write_extended_header() {
+        let data: ndarray::ArrayBase<ndarray::OwnedRepr<f64>, ndarray::Dim<[usize; 2]>> = Array2::zeros((8, 8));
```

Comment:
Why do you use this complex initialization? Isn't it the same as

    let data = Array2::<f64>::zeros((8, 8));
    or
    let data: Array2<f64> = Array2::zeros((8, 8));

Reply:
Its what the compiler said, and being not super familar with ndarray I simply obliged. Will fix.